### ISSUE TITLE
Upload Before We Start PDF

### ISSUE THREAD

**Issue by chauhankiran** [chauhankiran/msds#5]
Upload Before We Start PDF in pdfs folder.

**Comment by chauhankiran** [comment]
The plan is changed. We are going to create .html page for this instead of circulating pdf. So, deprecating this issue in favour of creating .html for Before We Start. There will be new ticket for this.
